modules/prefs.js

```diff
@@ -65,6 +65,7 @@ function PrefManager() {
   this.hide_quote_length = prefsService.getIntPref("hide_quote_length");
   this.hide_sigs = prefsService.getBoolPref("hide_sigs");
   this.compose_in_tab = prefsService.getBoolPref("compose_in_tab");
+  this.message_open_in_conversation = prefsService.getBoolPref("message_open_in_conversation");
```

Comment:
also I think you're missing something in this file to watch for preferences changes